doc/api/fs.md

```diff
@@ -5094,6 +5105,9 @@ changes:
 * `encoding` {string}
 * Returns: {number} The number of bytes written.
 
+If `string` is a plain object, like all variations of `writeFile`, it must have
```

Comment:
I'm not sure if this is applicable, but `Object` is listed in the types, so I guessed yes. Wanted to specifically call out just in case.